sdk/keyvault/azure-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/cryptography/AesCbcHmacSha2.java

```diff
@@ -78,6 +78,74 @@
 
             return inner.doFinal(input);
         }
+
+        private static byte[] toBigEndian(long i) {
```

Comment:
Use a descriptive param name instead of just `i`.

Reply:
Will refactor in a follow up PR, today.